lib/src/model/broadcast/broadcast.dart

```diff
@@ -46,15 +46,18 @@ class BroadcastRound with _$BroadcastRound {
   }) = _BroadcastRound;
 }
 
+typedef BroadcastMapGames = IMap<BroadcastGameId, BroadcastGameSnapshot>;
+
 @freezed
 class BroadcastGameSnapshot with _$BroadcastGameSnapshot {
   const BroadcastGameSnapshot._();
 
   const factory BroadcastGameSnapshot({
-    required IList<BroadcastPlayer> players,
+    required IMap<Side, BroadcastPlayer> players,
     required String fen,
     required Move? lastMove,
     required String status,
+    required Duration? thinkTime,
```

Comment:
What is the use of `thinkTime`?

Reply:
`thinkTime` is the amount of time that the player whose turn it is has been thinking since his last move

Reply:
> thinkTime is the amount of time that the player whose turn it is has been thinking since his last move.

Would be good to put that explanation as a field comment.